travvy/package.json

```diff
@@ -10,6 +10,7 @@
   "dependencies": {
     "axios": "^0.20.0",
     "core-js": "^3.6.5",
+    "firebase": "^8.1.1",
```

Comment:
I had to add this too as I was given an error when I tried to run the front end - this must be something extra we need in our package.json from the work Rushan has been doing with file upload!